python/requirements/ml/requirements_tune.txt

```diff
@@ -33,6 +33,7 @@ pytest-remotedata==0.3.2
 lightning-bolts==0.4.0
 protobuf==3.19.6
 pytorch-lightning==1.6.5
+fairscale==0.4.6
```

Comment:
Should this go in `requirements-train.txt` instead?

Reply:
I saw the comments in `requirements_train.txt`, saying
```
# TODO(train-team): Remove this once Ray Train is deprecated.
```
I believe this file will be deleted in the future so I put in requirements_tune.txt.

Reply:
Once Ray Train is deprecated 😳

@matthewdeng Any ideas about this?

Reply:
😨 @amogkam do you remember why this TODO was added [here](https://github.com/ray-project/ray/pull/22645/files)? Was it because we were planning to move it into AIR?

Reply:
yeah its just to consolidate with `requirements_tune.txt`

Reply:
is `fairscale` a necessary dependency? fsdp has been upstreamed into pytorch

Reply:
Yes It's necessary for backward compatibility.

Reply:
Oh would it be reasonable given the current state of the world to instead have `requirements_train.txt` include `requirements_tune.txt` plus train specific dependencies?

Reply:
> Yes It's necessary for backward compatibility.

backwards compatibility for what?

Reply:
For users with an older version of PyTorch (<1.11).

Reply:
> Oh would it be reasonable given the current state of the world to instead have requirements_train.txt include requirements_tune.txt plus train specific dependencies?

not sure if this is the right layering...but yes they should be more consolidated